Sources/Datadog/Tracing/Propagation/TracePropagationHeadersExctractor.swift

```diff
@@ -0,0 +1,12 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+public protocol TracePropagationHeadersExctractor {
```

Comment:
```suggestion
public protocol TracePropagationHeadersExtractor {
```

Nice 👍 can you please add documentation to both Extractor and Provider protocols?